web/crux/prisma/schema.prisma

```diff
@@ -525,3 +528,33 @@ model Storage {
 
   @@unique([name, teamId])
 }
+
+model ConfigBundle {
+  id        String   @id @default(uuid()) @db.Uuid
+  name      String
```

Comment:
Didn't we want to include a brief description for these?

Reply:
I'm not sure, but I can add a simple description field easily.

Reply:
Yeah, please.